spec.html

```diff
@@ -31484,7 +31484,7 @@ <h1>Runtime Semantics: RepeatMatcher ( _m_, _min_, _max_, _greedy_, _x_, _c_, _p
             <p>because each iteration of the outermost `*` clears all captured Strings contained in the quantified |Atom|, which in this case includes capture Strings numbered 2, 3, 4, and 5.</p>
           </emu-note>
           <emu-note>
-            <p>Step 1 of the RepeatMatcher's _d_ closure states that, once the minimum number of repetitions has been satisfied, any more expansions of |Atom| that match the empty character sequence are not considered for further repetitions. This prevents the regular expression engine from falling into an infinite loop on patterns such as:</p>
+            <p>Step a of the RepeatMatcher's _d_ closure states that, once the minimum number of repetitions has been satisfied, any more expansions of |Atom| that match the empty character sequence are not considered for further repetitions. This prevents the regular expression engine from falling into an infinite loop on patterns such as:</p>
```

Comment:
This one's kind of weird. Step 2 defines the closure, so Step 2.a would therefore be "Step a of the ... closure".

The thing is that in [ES2015](https://www.ecma-international.org/ecma-262/6.0/#sec-runtime-semantics-repeatmatcher-abstract-operation), we actually ensured that this Step 2 in particular had Arabic numeral substeps, perhaps for this very purpose. But this was lost as of [ES2016](https://www.ecma-international.org/ecma-262/7.0/#sec-runtime-semantics-repeatmatcher-abstract-operation).

If "Step a" is too weird I can try to rephrase such that Step 2.a will be appropriate.

Reply:
this one is kind of weird. should we perhaps force the closure steps to be numeric again?

Reply:
Do you know if there's a simple way to do that in ecmarkup? Seems like just adding an extra `<emu-alg>` layer in there doesn't help.

Reply:
hm, doesn't look like ecmarkup has that option - cc @rbuckton?

Reply:
I think "Step 2.a of RepeatMatcher" would be fine. (The fact that it's a step of a closure created by RepeatMatcher isn't vital to what the Note is saying.)